sdk/schemaregistry/schema-registry-avro/src/schemaRegistryAvroSerializer.ts

```diff
@@ -151,9 +151,8 @@ export class SchemaRegistryAvroSerializer {
   //
   // Avoid hitting the schema registry on every call by caching schemas.
   //
-  // Currently the cache can only be discarded by throwing away the serializer
-  // and creating a new one, but this will be revisited after the initial
-  // preview:
+  // Currently the serializer-level cache can only be discarded by throwing away the serializer
+  // and creating a new one, but this will be revisited before the general availability release
```

Comment:
This now references an issue that's going to be closed by this change below. Maybe we should just document that the serializer also holds on to things for its lifetime and remove this comment?

Reply:
I deleted the comment and added a note to the readme. Please let me know if that looks good to you.